tests/x11regressions/firefox/sle12/firefox_java.pm

```diff
@@ -22,9 +22,10 @@ sub java_testing {
     send_key "alt-d";
     type_string "http://www.java.com/en/download/installed.jsp?detect=jre\n";
 
-    assert_screen [qw(firefox-extcontent-reader firefox-java-security oracle-cookies-handling)];
-    if (match_has_tag 'firefox-extcontent-reader') {
-        assert_and_click('firefox-extcontent-reader');
+    wait_still_screen;
```

Comment:
the default timeout for wait_still_screen is 7 seconds. Do we really need to wait these 7 seconds every time?